src/main/resources/sample-config.yml

```diff
@@ -3,6 +3,8 @@
 # The command prefixes
 prefixes: ["!", "?"]
 botToken: "your token"
+# the url for the database connection
+dburl: "jdbc:sqlite:yourfile.sqlite"
```

Comment:
Enter the same db url as in the pom. Since it did not seem to work with 2 different urls.